src/server_manager/install_scripts/do_install_server.sh

```diff
@@ -147,13 +149,13 @@ export SB_PUBLIC_IP=$(cloud::public_ip)
 
 log_for_sentry "Initializing ACCESS_CONFIG"
 export ACCESS_CONFIG="$SHADOWBOX_DIR/access.txt"
-> $ACCESS_CONFIG
+true > $ACCESS_CONFIG
 
 # Set trap which publishes an error tag and sentry report only if there is an error.
 function finish {
   INSTALL_SERVER_EXIT_CODE=$?
   log_for_sentry "In EXIT trap, exit code $INSTALL_SERVER_EXIT_CODE"
-  if [[ -z $(grep apiUrl $ACCESS_CONFIG) ]] || [[ -z $(grep certSha256 $ACCESS_CONFIG) ]]; then
+  if ! ( grep -q apiUrl $ACCESS_CONFIG && grep -q certSha256 $ACCESS_CONFIG ); then
```

Comment:
Perhaps prefer `--quiet` for readability?

Reply:
Done